queries.sql

```diff
@@ -5,4 +5,82 @@ SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = '1';
 SELECT * from animals WHERE name != 'Gabumon';
-SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+begin
+
+UPDATE animals
+SET species = 'unspecified'
+ROLLBACK
+
```

Comment:
- [ ] Good job making the changes as requested here to `UPDATE` the animals table by setting the species column to unspecified, but as mentioned earlier for this step you are explicitly expected to verify that change was made and rollbacked, so make sure that you included the `SELECT species from animals;` before and after the `ROLLBACK` statement.